web/js/search.js

```diff
@@ -0,0 +1,43 @@
+$(document).ready(function () {
```

Comment:
Don't forget documentation.  Also, preferably use the [Module pattern](http://www.adequatelygood.com/JavaScript-Module-Pattern-In-Depth.html) to prevent pollution of the global scope and accidental namespace collusions.